languages/python/oso/tests/test_polar_roles.py

```diff
@@ -338,10 +332,10 @@ def test_nested_dot_relationship(init_oso):
     oso.load_str(policy)
 
     with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
+        oso.validate_config()
 
 
-@pytest.mark.skip("TODO: validation")
+@pytest.mark.skip("can't detect, not sqlalchemy")
```

Comment:
let's either kill this or update the message to mirror the reason we can't detect it
```suggestion
@pytest.mark.skip("Cannot generically walk fields of a Python class")
```